lib/tldr.js

```diff
@@ -15,16 +15,12 @@ const exit = process.exit;
 exports.list = (singleColumn) => {
   let os = platform.getPreferredPlatformFolder();
   index.commandsFor(os)
-    .then((commands) => {
-      printPages(commands, singleColumn);
-    });
+    .then((commands) => printPages(commands, singleColumn));
```

Comment:
Please use the full `() => {}` syntax as mentioned earlier.

Reply:
Why here? Are you trying to completely avoid short form? 🤔

Reply:
Yes, that's right.

Reply:
Huh, luckily my whole PR is (not) going in opposite direction 😿 
I'll need more time to address all occurrences... 

Also this is good candidate for eslint config update: https://eslint.org/docs/rules/arrow-body-style

Reply:
Since you just merged #200 I'll merge master here to get ESLint guide me 💯 🏄

Reply:
Yep, that was the intention :laughing: